traffic_ops/traffic_ops_golang/routing/routes.go

```diff
@@ -279,6 +280,11 @@ func Routes(d ServerData) ([]Route, []RawRoute, http.Handler, error) {
 		{api.Version{2, 0}, http.MethodPost, `regions/?$`, api.CreateHandler(&region.TORegion{}), auth.PrivLevelOperations, Authenticated, nil, 2288334488, noPerlBypass},
 		{api.Version{2, 0}, http.MethodDelete, `regions/?$`, api.DeleteHandler(&region.TORegion{}), auth.PrivLevelOperations, Authenticated, nil, 2232626758, noPerlBypass},
 
+		{api.Version{2, 0}, http.MethodPost, `topologies/?$`, api.CreateHandler(&topology.TOTopology{}), auth.PrivLevelOperations, Authenticated, nil, 3871452221, noPerlBypass},
```

Comment:
I'm starting to think these need to all be 3.0 as it looks like the 4.1 ATC branch is about to be cut and the next TC version will be 5.0 that will have APIv3....

Reply:
That is probably correct, but I think there may be some overhead involved in getting API v3 up and running that should probably be its own PR.